gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -214,10 +219,14 @@ private void setupErrorProneOptions(
                 @Override
                 public Iterable<String> asArguments() {
                     List<String> suppressionsToRemove = checksToRemoveSuppressionsFor(project);
+                    Set<String> checksToPatch = new HashSet<>();
+                    checksToPatch.add("RemoveRolloutSuppressions");
+                    // TODO(aldexis): Should we only do this if a specific argument is present?
+                    checksToPatch.addAll(suppressionsToRemove);
```

Comment:
I'm wondering whether I should make a new argument and/or reuse errorProneApply (potentially with an extra check that you can't specify specific different checks to patch if doing the remove suppressions as well, since we should only patch these)